tests/transports-governance/test_governance_edge_cases_comprehensive.py

```
@@ -0,0 +1,1433 @@
+
+        for op in operations:
+            try:
+                result = op()
+                results.append(result)
+
+                # Track created entities
+                if hasattr(result, "status_code") and result.status_code == 201:
+                    try:
+                        entity_id = result.json()["id"]
+                        track_created_entity("customers", entity_id)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)